.gitignore

```diff
@@ -5,3 +5,4 @@
 /vendor/
 /zf-mkdoc-theme.tgz
 /zf-mkdoc-theme/
+/.idea/
```

Comment:
This should be removed - please update your global gitignore. Within the project we shouldn't have any IDE/OS specific files/directories.